src/main/kotlin/org/jitsi/jicofo/xmpp/XmppServices.kt

```diff
@@ -80,3 +99,62 @@ class XmppServices(xmppProviderFactory: XmppProviderFactory) {
 }
 
 enum class XmppConnectionEnum { Client, Service }
+
+class AvModerationHandler(val xmppConnection: AbstractXMPPConnection) : StanzaListener, RegistrationListener {
+    var conferenceStore: ConferenceStore = EmptyConferenceStore()
+    private val jsonParser = JSONParser()
+    private var avModerationAddress: String? = null
+
+    override fun processStanza(stanza: Stanza) {
+        if (!stanza.from.equals(avModerationAddress)) {
```

Comment:
`stanza.from != avModerationAddress`? Can `stanza.from` ever be `null`?

Reply:
I don't think so, but I will add the check.
using equals is intentional as it is overloaded equals of DomainPartJid comparing with a string. (kotlin gives an error with !=)

Reply:
Ah okay. Why do we keep the address as a String then?

Reply:
Ok, I will make it jid ... it used to be jid but I move it several times while debugging an issue